src/counter/counterReducer.ts

```diff
@@ -0,0 +1,39 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { CounterState } from "../types/redux-types";
+import { incrementAsync } from "./counterActions";
+const initialState: CounterState = {
+  value: 0,
+};
+
+const counterReducer = createSlice({
+  name: "counter",
+  initialState,
+  reducers: {
+    increment: (state) => {
```

Comment:
The state actions are not specced in the ticket. We just need the initial state of user, instead of a counterState.